patches/tModLoader/Terraria/Social/Steam/WorkshopHelper.TML.cs

```diff
@@ -108,7 +109,11 @@ internal class ModManager
 
 				File.WriteAllText(apptxt, thisApp.ToString());
 
-				if (GameServer.Init(0x7f000001, 7775, 7774, EServerMode.eServerModeNoAuthentication, "0.11.9.0")) {
+				if (RuntimeInformation.ProcessArchitecture is Architecture.Arm or Architecture.Arm64) {
```

Comment:
probably just `Architecture.Arm64` - `Architecture.Arm` is AArch32 which it doesn't seem this patch is targeting